### PR TITLE
Clean up Windows IBrowser implementations

### PR DESCRIPTION
**Pull request by damieng**

### CODE REVIEW

File: src/Auth0.OidcClient.WPF/PlatformWebView.cs [hunk 1]
```diff
@@ -43,64 +43,52 @@ public PlatformWebView(string title = "Authenticating ...", int width = 1024, in
...
-                    ResultType = BrowserResultType.UserCancel
-                };
-
                 window.Closed += (o, e) =>
                 {
                     signal.Release();
                 };
-                browser.LoadCompleted += (sender, args) =>
+
+                browser.LoadCompleted += (s, e) =>
```
Comment (luisrudge) [inline src/Auth0.OidcClient.WPF/PlatformWebView.cs]: why change the argument name? it was very clear what they were before

File: src/Auth0.OidcClient.Core/Auth0ClientBase.cs [hunk 2]
```diff
@@ -119,7 +119,7 @@ private OidcClientOptions CreateOidcClientOptions(Auth0ClientOptions options)
                 Flow = AuthenticationFlow.AuthorizationCode,
                 ResponseMode = _formPost ? AuthorizeResponseMode.FormPost : AuthorizeResponseMode.Redirect,
                 RedirectUri = options.RedirectUri ?? $"https://{_options.Domain}/mobile",
-                PostLogoutRedirectUri = options.PostLogoutRedirectUri,
+                PostLogoutRedirectUri = options.PostLogoutRedirectUri ?? $"https://{_options.Domain}/mobile",
```
Comment (luisrudge) [inline src/Auth0.OidcClient.Core/Auth0ClientBase.cs]: should we add a test for this case?

File: src/Auth0.OidcClient.WPF/PlatformWebView.cs (hunk 1 above)
Comment (damieng) [reply src/Auth0.OidcClient.WPF/PlatformWebView.cs]: Event handler args are always named 'e' on .NET. It's a standard practice that even Microsoft promotes e.g. https://docs.microsoft.com/en-us/dotnet/api/system.eventhandler?view=netframework-4.8

I could revert sender back if you prefer - that practice is less well-spread.

File: src/Auth0.OidcClient.Core/Auth0ClientBase.cs (hunk 2 above)
Comment (damieng) [reply src/Auth0.OidcClient.Core/Auth0ClientBase.cs]: There are no unit tests today for this library.

File: src/Auth0.OidcClient.WPF/PlatformWebView.cs (hunk 1 above)
Comment (luisrudge) [reply src/Auth0.OidcClient.WPF/PlatformWebView.cs]: nope, e is fine!

**Pull request merged**